### DISCUSSION TITLE
Time for a new release?

### DISCUSSION THREAD

**Discussion by Lingxi-Li** [photoview/photoview#880] [General]
It has been more than a year since last release. Maybe, it's time for a new release to catch the bugfixes and improvements made in the year?

**Comment by kkovaletp** [comment]
@viktorstrate could you please review open PRs and check reported issues, providing your feedback, thoughts, enhancements, and fixes?

**Comment by Lingxi-Li** [comment]
I took a look at the [tags](https://hub.docker.com/r/viktorstrate/photoview/tags) on Docker Hub. `latest` was pushed a year ago, while `master` was pushed a month ago. Looks like `master` is the tag to go if you want the latest version. The tag used in [example docker compose file](https://github.com/photoview/photoview/blob/master/docker-compose.example.yml#L16) is `2`, which does not make much sense to me. It should be either `latest` or `master`.

**Reply by kkovaletp** [reply]
the `master` tag simply reflects the state of the master branch in the repo, which is the main development branch for all new merges. You technically could use it in your compose, but you need to understand clearly that this is not yet released version, which might be not stable and potentially contain more bugs.

there is a better docker compose setup in my PR #863 and you could try it instead, but it also uses the `latest` tag

**Reply by Lingxi-Li** [reply]
Yup. Version info is not embedded/shown for `master` tag.

![image](https://github.com/photoview/photoview/assets/11997203/ad0b5078-9075-4bd3-8e16-d447faaea7ba)

**Comment by kkovaletp** [comment]
I've started the new discussion #893, related to this topic. Please check it)

**Comment by Lord0fBytes** [comment]
I checked out post #893 and it looks like there are some new maintainers. Will there be new releases going forward? I am still only seeing the `2.3.13` version from 2 years ago.

**Reply by kkovaletp** [reply]
@Lord0fBytes, yes, there are maintainers and we're working to make the release happen soon.
There are a few PRs to be finalized and merged before the release, but I hope that it can be done relatively fast.

Until that feel free to use the `master` image (if you use docker, make sure to follow the new configuration steps from the readme in this casse) or branch (if you host it directly on your system)

**Reply by Lord0fBytes** [reply]
This is great to hear. I really love this app and have been looking for something open source to help out with. Glad this is still going strong and cant wait for help out with it.

**Reply by kkovaletp** [reply]
There are many ways to contribute to our project)
We're still working on the Contribution guide, but you can start by looking into the current draft of it: #943. Then, depending on your skills and free time, you can choose the best for you way of contribution and start the process)
You (and everybody else) are welcome to join the active contributors group) Looking forward to working with you)
